### ISSUE TITLE
Update engine to node >18 for package.json

### ISSUE THREAD

**Issue by klein0r** [ioBroker/ioBroker#535]
https://github.com/ioBroker/ioBroker/blob/2cf859a3a7b4e3414e4e01efe6d5e045b57e7e81/installer.sh#L128-L134

**Comment by Apollon77** [comment]
Yes this is planned as soon as js.controller 6 hits stable ... for now we still install js.controller 5 with the installer by default which supports node-js 16

**Comment by Apollon77** [comment]
done by installer release today
